packages/cli/src/commands/exchange/gold.ts

```diff
@@ -35,9 +35,12 @@ export default class ExchangeGold extends BaseCommand {
     const goldToken = await this.kit.contracts.getGoldToken()
     const exchange = await this.kit.contracts.getExchange()
 
-    await displaySendTx('approve', goldToken.approve(exchange.address, sellAmount.toFixed()))
```

Comment:
it seems like before you were calling both approve and increaseAllowance but they seem to both update the same `allowance` field.  so wouldn't the allowance become twice as much?  why would we get an error that says the allowance isn't high enough?